shared/cluster.go

```diff
@@ -655,3 +656,64 @@ func PrintGetAll() {
 	fmt.Printf("\n\n\n-----------------  Results from kubectl get all -A -o wide"+
 		"  -------------------\n\n%v\n\n\n\n", res)
 }
+
+func CreateSecret(secret, namespace string) error {
```

Comment:
if am using this function somewhere else, and forget to send secret or send a empty string, is that ok ? 
or should we validate that ?